### DISCUSSION TITLE
Release date for helmfile/vals hotfix

### DISCUSSION THREAD

**Discussion by SpyrosMoux** [helmfile/helmfile#1073] [Questions] [answered]
Hello everyone, 

I had an issue with my helmfile, similar to this [https://github.com/fujiwara/tfstate-lookup/issues/139](https://github.com/fujiwara/tfstate-lookup/issues/139) which has since been patched. In the go.mod I see the that the vals dependency has been updated to the corrected version 
```
github.com/helmfile/vals v0.28.1
```
When are we getting a new release that will have this change?

Thank you in advance.

**Comment by yxxhero** [comment, marked as answer]
@SpyrosMoux this week

**Reply by SpyrosMoux** [reply]
Perfect thank you
